frontend/src/reducers/windowHandler.js

```diff
@@ -314,10 +314,10 @@ export const getMasterDocStatus = createSelector(getData, (data) => {
 
 export default function windowHandler(state = initialState, action) {
   switch (action.type) {
-    case NO_CONNECTION:
+    case CONNECTION_ERROR:
       return {
         ...state,
-        connectionError: action.status,
+        connectionError: action.errorType,
```

Comment:
rename `connectionError` to `errorType`.

Reply:
done